k8s/aat/common/bsp/bulk-scan-processor.yaml

```diff
@@ -25,6 +36,45 @@ spec:
       image: hmctspublic.azurecr.io/bulk-scan/processor:prod-72f18417
       environment:
         STORAGE_BLOB_SELECTED_CONTAINER: ALL
+      testsConfig:
+        memoryLimits: "3072Mi"
+        serviceAccountName: tests-service-account
+        keyVaults:
+          bulk-scan:
+            resourceGroup: bulk-scan
+            secrets:
+              storage-account-primary-key: TEST_STORAGE_ACCOUNT_KEY
+              storage-account-name: TEST_STORAGE_ACCOUNT_NAME
+              processed-envelopes-queue-send-connection-string: PROCESSED_ENVELOPES_QUEUE_WRITE_CONN_STRING
+          s2s:
+            resourceGroup: rpe-service-auth-provider-aat
+            secrets:
+              microservicekey-bulk-scan-processor-tests: TEST_S2S_SECRET
+        environment:
+          TEST_URL: "http://bulk-scan-processor-java"
+          SLACK_CHANNEL: "bsp-build-notices"
+          SLACK_NOTIFY_SUCCESS: true
+          TEST_S2S_NAME: "bulk_scan_processor_tests"
+          TEST_S2S_URL: "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
+          TEST_SCAN_DELAY: "4000"
+          TEST_STORAGE_CONTAINER_NAME: "bulkscan"
+          TEST_STORAGE_ACCOUNT_URL: "https://bulkscan.aat.platform.hmcts.net"
+          PROXYOUT_ENABLED: true
+          RUN_END_TO_END_TESTS: true
+      smoketestscron:
+        enabled: true
+        image: hmctspublic.azurecr.io/bulk-scan/processor-test:prod-1795cc55
+        schedule: "0/30 * * * *"
+      functionaltestscron:
+        enabled: true
+        image: hmctspublic.azurecr.io/bulk-scan/processor-test:prod-1795cc55
+        schedule: "15/30 * * * *"
```

Comment:
These two will go away later, but this is to verify if scheduled execution works as expected.